### PR TITLE
feat: now listing customer cloud in rhoas kafka list

### PR DESCRIPTION
**Pull request by jackdelahunt**
Closes # 

### Verification Steps

1. Have an enterprise cluster
2. `rhoas kafka list` you will see the customer cloud info listed

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: pkg/cmd/kafka/list/list.go [hunk 1]
```diff
@@ -160,13 +161,22 @@ func mapResponseItemsToRows(kafkas []kafkamgmtclient.KafkaRequest, selectedId st
 		if k.GetId() == selectedId {
 			name = fmt.Sprintf("%s %s", name, icon.Emoji("✔", "(current)"))
 		}
+
+		var customerCloud string
+		if id, ok := k.GetClusterIdOk(); ok {
+			customerCloud = *id
+		} else {
+			customerCloud = "Redhat Infrastructure"
```
Comment (machi1990) [inline pkg/cmd/kafka/list/list.go:169]:
```suggestion
			customerCloud = "Red Hat Infrastructure"
```
should it be?

File: pkg/cmd/kafka/list/list.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ type kafkaRow struct {
 	Status        string `json:"status" header:"Status"`
 	CloudProvider string `json:"cloud_provider" header:"Cloud Provider"`
 	Region        string `json:"region" header:"Region"`
+	CustomerCloud string `json:"customer_cloud" header:"CustomerCloud"`
```
Comment (machi1990) [inline pkg/cmd/kafka/list/list.go:37]: Was this the agreed name?

Comment (machi1990) [reply pkg/cmd/kafka/list/list.go:37]: If not already agreed, let's revisit this by using the agreed header name.

Comment (machi1990) [reply pkg/cmd/kafka/list/list.go:37]: Also the header has to be `Customer Cloud` if we stick with that name:-)

**Review by machi1990 — CHANGES_REQUESTED** [review]
Left naming suggestions.

If the names are not agreed at the moment, let's revisit them at some point.

File: pkg/cmd/kafka/list/list.go (hunk 2 above)
Comment (machi1990) [inline pkg/cmd/kafka/list/list.go:37]:
```suggestion
	CustomerCloud string `json:"customer_cloud" header:"Customer Cloud"`
```

Comment (jackdelahunt) [reply pkg/cmd/kafka/list/list.go:37]: We have not officially agreed on a name for the output, @dimakis what do you think?

Comment (dimakis) [reply pkg/cmd/kafka/list/list.go]: There is no agreed names as of yet, I've brought this up a couple of times at meetings but so far we don't have names for `Red Hat OpenShift Cluster` or  `Your OpenShift Cluster`

Comment (jackdelahunt) [reply pkg/cmd/kafka/list/list.go]: From those two I think `Red Hat OpenShift Cluster` sounds good to me

Comment (machi1990) [reply pkg/cmd/kafka/list/list.go]: Let's go with what we've hear and raise it in the sync call.

**Pull request merged**